.github/workflows/ruby.yml

```diff
@@ -5,27 +5,35 @@
 # This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
 # For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
 
-name: Ruby
+name: Test Suite
 
 on:
   push:
-    branches: [ $default-branch ]
   pull_request:
     branches: [ $default-branch ]
 
 permissions:
   contents: read
 
 jobs:
-  test:
+  rspec:
 
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['3.0']
+        ruby-version: ['3.2.0']
```

Comment:
I read somewhere we don't need this if we have `.ruby-version` which we do.  Can you try removing so we don't have it specified in two places?

Reply:
yup you were right. didn't need it.